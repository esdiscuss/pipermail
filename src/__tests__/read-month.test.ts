import readMonth from '../read-month';

test('parses a month listing page to return an array of message urls', async () => {
  const res = await readMonth(
    'https://mail.mozilla.org/pipermail/es-discuss/2011-December',
  );
  expect(Array.isArray(res)).toBeTruthy();
  expect(res.length).toBeGreaterThan(10);
  res.forEach(url => {
    expect(url).toMatch(
      /^https:\/\/mail.mozilla.org\/pipermail\/es-discuss\/2011-December\/\d\d\d\d\d\d\.html$/,
    );
  });
  expect(res).toMatchSnapshot();
});
