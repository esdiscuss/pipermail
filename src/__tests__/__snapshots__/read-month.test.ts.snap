// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parses a month listing page to return an array of message urls 1`] = `
Array [
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018815.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018816.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018817.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018818.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018819.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018820.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018821.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018822.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018823.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018824.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018825.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018826.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018828.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018827.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018829.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018830.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018831.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018832.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018833.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018834.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018835.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018836.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018837.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018838.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018839.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018840.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018841.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018842.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018843.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018844.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018847.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018846.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018845.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018848.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018849.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018850.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018851.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018852.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018853.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018854.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018855.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018856.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018857.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018858.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018859.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018860.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018861.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018862.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018863.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018864.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018865.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018866.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018867.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018868.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018869.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018870.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018871.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018872.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018873.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018874.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018875.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018876.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018877.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018878.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018879.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018880.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018881.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018882.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018883.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018884.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018885.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018886.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018887.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018888.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018889.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018890.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018891.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018892.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018893.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018894.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018895.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018896.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018897.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018898.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018899.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018900.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018901.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018902.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018903.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018904.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018905.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018906.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018907.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018908.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018909.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018910.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018911.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018912.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018913.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018914.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018915.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018916.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018917.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018918.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018919.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018920.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018921.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018922.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018923.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018924.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018925.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018926.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018927.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018928.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018929.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018930.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018931.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018932.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018933.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018934.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018935.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018936.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018937.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018938.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018940.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018939.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018941.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018942.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018943.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018944.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018945.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018946.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018947.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018948.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018949.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018950.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018951.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018952.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018953.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018954.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018955.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018956.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018957.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018958.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018959.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018960.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018961.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018962.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018963.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018964.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018965.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018966.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018967.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018968.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018969.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018970.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018971.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018972.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018973.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018974.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018975.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018976.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018977.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018978.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018980.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018979.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018981.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018982.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018983.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018984.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018985.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018986.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018987.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018988.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018989.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018990.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018991.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018992.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018993.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018994.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018995.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018996.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018997.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018998.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/018999.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019000.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019001.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019002.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019003.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019004.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019005.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019006.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019007.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019008.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019009.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019010.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019011.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019012.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019013.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019014.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019015.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019016.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019017.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019018.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019019.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019020.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019021.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019022.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019023.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019024.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019025.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019026.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019027.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019028.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019029.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019030.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019031.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019032.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019033.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019034.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019035.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019036.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019037.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019038.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019039.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019040.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019041.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019042.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019043.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019044.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019045.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019046.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019047.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019048.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019049.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019050.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019051.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019052.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019053.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019054.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019055.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019056.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019058.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019057.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019059.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019060.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019061.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019062.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019063.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019064.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019065.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019066.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019067.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019068.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019069.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019070.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019071.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019072.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019073.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019074.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019075.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019076.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019077.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019078.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019079.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019080.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019081.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019082.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019083.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019084.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019085.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019086.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019087.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019088.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019090.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019089.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019091.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019092.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019093.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019094.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019095.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019096.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019097.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019098.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019099.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019100.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019101.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019102.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019103.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019104.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019105.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019106.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019107.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019108.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019109.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019110.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019111.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019112.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019113.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019114.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019115.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019116.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019117.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019118.html",
  "https://mail.mozilla.org/pipermail/es-discuss/2011-December/019119.html",
]
`;
